packages/run-it/src/components/CopyClipboardIconButton/CopyClipboardIconButton.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import React, { FC, useRef, useState } from 'react'
+import { IconButton } from '@looker/components'
+import { Done } from '@styled-icons/material/Done'
+import { Assignment } from '@styled-icons/material/Assignment'
+
+export interface CopyClipboardIconButtonProps {
+  content: string
+  tooltip?: string
+  success?: string
+}
+
+export const CopyClipboardIconButton: FC<CopyClipboardIconButtonProps> = ({
+  content,
+  tooltip = 'Copy to clipboard',
+  success = 'Copied',
+}) => {
+  const [copied, setCopied] = useState(false)
+  const buttonRef = useRef<HTMLButtonElement>(null)
+
+  const clickCopyButton = () => {
+    const textField = document.createElement('textarea')
+    textField.value = content
+
+    if (buttonRef.current) {
+      buttonRef.current.appendChild(textField)
+      textField.select()
+      document.execCommand('copy')
+      textField.remove()
+      setCopied(true)
+      setTimeout(() => setCopied(false), 2500)
```

Comment:
My initial thought was ugh but with the label indicating success I guess okay. I assume the label shows up as a tooltip because the user is in the process of clicking

Reply:
This is based on the `@looker/components` version still being used in `CodeCopy`. I haven't figured out how to do this right, yet, so this Component is a WIP until I learn how to do it correctly.

Reply:
Removed this because of `a11y` issues